tutorials/26_Hybrid_Retrieval.ipynb

```diff
@@ -0,0 +1,473 @@
+    "pipeline = Pipeline()\n",
+    "pipeline.add_node(component=sparse_retriever, name=\"SparseRetriever\", inputs=[\"Query\"])\n",
+    "pipeline.add_node(component=dense_retriever, name=\"DenseRetriever\", inputs=[\"Query\"])\n",
+    "pipeline.add_node(component=join_documents, name=\"JoinDocuments\", inputs=[\"SparseRetriever\", \"DenseRetriever\"])\n",
+    "pipeline.add_node(component=rerank, name=\"ReRanker\", inputs=[\"JoinDocuments\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "V3bsFkHuhHn4"
+   },
+   "source": [
+    "### Generating a Pipeline Diagram\n",
+    "\n",
+    "With any Pipeline, whether prebuilt or custom constructed, you can save a diagram showing how all the components are connected."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "oCIMtwmThQG4"
+   },
+   "outputs": [],
+   "source": [
+    "# Uncomment the following to generate the images\n",
+    "# !apt install libgraphviz-dev\n",
+    "# !pip install pygraphviz\n",
+    "\n",
+    "# pipeline.draw(\"pipeline_hybrid.png\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "sTTVLUJylXRx"
+   },
+   "source": [
+    "Search and article with Hybrid Retrieval, if you want to see all the steps you can enable `debug=True` in `JoinDocuments`'s `params`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "p-5WbeBulXR0"
+   },
+   "outputs": [],
+   "source": [
+    "prediction = pipeline.run(\n",
+    "    query=\"treatment for HIV\",\n",
+    "    params={\n",
+    "        \"SparseRetriever\": {\"top_k\": 10},\n",
+    "        \"DenseRetriever\": {\"top_k\": 10},\n",
+    "        \"JoinDocuments\": {\"top_k_join\": 15},  # comment for debug\n",
+    "        # \"JoinDocuments\": {\"top_k_join\": 15, \"debug\":True}, #uncomment for debug\n",
+    "        \"ReRanker\": {\"top_k\": 5},\n",
+    "    },\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "WvPv1cJ6gbBJ"
+   },
+   "source": [
+    "Let's create a function to print a kind of *search page*."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "raL_z_sByDoQ"
+   },
+   "outputs": [],
+   "source": [
+    "def pretty_print_results(prediction):\n",
+    "    for doc in prediction[\"documents\"]:\n",
+    "        print(doc.meta[\"title\"], \"\\t\", doc.score)\n",
+    "        print(doc.meta[\"abstract\"])\n",
+    "        print(\"\\n\", \"\\n\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "mSUiizGNytwX"
+   },
+   "outputs": [],
+   "source": [
+    "pretty_print_results(prediction)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "rXxhhdQulXR5"
+   },
+   "source": [
+    "## About us\n",
+    "\n",
+    "*Leave this section as is. It's a footer that we add to all our tutorials.*\n",
```

Comment:
@bilgeyucel I'm not entirely sure if this line should stay here?

Reply:
@nickprock you can just remove the "About Us" section completely

Reply:
Done